src/Microsoft.Management.Deployment.InProc/dllmain.cpp

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+#include "pch.h"
+#include <WindowsPackageManager.h>
+
+extern "C"
+{
+    BOOL WINDOWS_PACKAGE_MANAGER_API_CALLING_CONVENTION DllMain(

```

Comment:
I think we should be using a macro for calling convention that is more in line with what this expects, not just whatever we happen to use.